### ISSUE TITLE
JabFox: Error connecting to JabRef:  <unavailable>

### ISSUE THREAD

**Issue by Pratched** [JabRef/JabRef-Browser-Extension#56]
hey there I have a similar problem to #50.

Using Win10 Firefox 64.2 (64Bit) with Jabfox and JabRef 4.3 I receive the stated error when hitting the provided JabFox-Button in Firefox.

```
JabFox: Error connecting to JabRef:  <unavailable> connector.js:76:5
     Zotero.Connector</this.sendBibTexToJabRef/< moz-extension://4d844784-ebb7-4319-869c-da481277a621/connector.js:76:5
```
I have double-checked the reg-entry and the FF-addon-page says "Communication to JabRef successfull". 
PowerShell Version 5.1
JabRef-"remote operation" is checked.

Any ideas?

**Comment by tobiasdiez** [comment]
This error points to a problem with the registry setup. Can you please check that the registry looks as described in https://github.com/JabRef/JabFox/issues/50#issuecomment-379888605

**Comment by Pratched** [comment]
Yes I checked and it does. The path points to the only `JabRef.json` I could find.


![grafik](https://user-images.githubusercontent.com/19487828/51771496-742bb100-20e9-11e9-96a4-a175e248b7ec.png)

**Comment by AEgit** [comment]
JabRef 5.0.0-dev--2019-10-21----0f0dce621
Windows 10 10.0 amd64
Java 12.0.2

JabFox 2.1.2
Firefox 70.0 (64-bit)

Cannot confirm. JabFox works as expected for me.

**Comment by tobiasdiez** [comment]
This should be fixed with the new version that will be released simultaneously with JabRef 5.0.
